### ISSUE TITLE
[Suggestion] Decal rendering distance 10x increase

### ISSUE THREAD

**Issue by Kaiser-Indrasil** [tsunamayo/Starship-EVO#4985]
I've noticed that decals can disappear really quickly when a player moves away from them and the reason for that is their small rendering distance.

At 50 degree FOV which (in my perception) is as natural to a human vision as I could get, the **10 times** increase of their rendering distance across the board would make them disappear much more seamlessly.

If it sounds like a radical change, I'd say let's use the Experimental branch and see what's the player reception - most importantly, if people's PCs are not burning down because of this proposed change.

If so, it can also be introduced as an additional slider in the Graphics settings.

https://user-images.githubusercontent.com/63756057/200087070-59d2fe08-992f-4228-ba73-ebe30d6d149d.mp4

![Starship_EVO_cEDGcsH9Lz](https://user-images.githubusercontent.com/63756057/200086364-97fe69e2-7435-45d5-84fd-1577f7c2abaa.jpg)
![Starship_EVO_FCAjIM29JS](https://user-images.githubusercontent.com/63756057/200086372-a7bc0648-234b-4365-bbab-b88401496cbc.jpg)
![Starship_EVO_FNN16TJYVL](https://user-images.githubusercontent.com/63756057/200086862-babbb098-8c53-429f-b414-2b6cbf1c7451.jpg)
![Starship_EVO_GYyc5laQPR](https://user-images.githubusercontent.com/63756057/200086377-c6cf2fe1-ac1a-4e10-a2e1-3f74542ef28b.jpg)

**Comment by kingvor** [comment]
yeah this would really be good, its such a pain decaling your ship to then it not been seen 90% of the time

**Comment by Horkas** [comment]
In my personal opinion, the best solution would be to just add a slider bar in the options for the draw distance of decals.

But if that's too much work, then doubling the current draw distance would be fine. But 10x it? No, sorry, but that will just tank performance too greatly. The two biggest factors that tank performance in a game is Calculations per Second, (Aka: Physics systems, game logic scripting, etc) and Texture draw.

With how obsessive tsunamayo is about optimization, (For good reason.), taking such a hit to performance for something that has no effect on physical gameplay is just not a good enough trade of the game engines available resources.

**Comment by Kaiser-Indrasil** [comment]
@Horkas I agree, a slider in the Graphics settings would be the best option.
